config/application.rb

```diff
@@ -33,3 +33,12 @@ class Application < Rails::Application
     config.api_only = true
   end
 end
+
```

Comment:
Oh this is interesting - is this there to solve a problem, or to prevent us from accidentally adding these bits in?

Reply:
Good question, I saw something similar to this in the GraphQL implementation walk through.